cmd/containerd/command/main.go

```diff
@@ -130,6 +132,19 @@ can be used and modified as necessary as a custom configuration.`
 			return err
 		}
 
+		// Initialize openTelemetry tracing
+		shutdown, err := tracing.InitOpenTelemetry(config)
+		log.G(ctx).Debug("Initializing opentelemetry")
+		if err != nil {
+			errors.Wrap(err, "failed to initialize openTelemetry tracing")
+		}
+		defer shutdown()
+
+		// Get tracer
+		ctrdTracer := otel.Tracer("containerd")
+		ctx, mainCtrdSpan := ctrdTracer.Start(gocontext.Background(), "containerd-expoerter")
+		defer mainCtrdSpan.End()
```

Comment:
In tracing, is it common to have the main span that lives while the server itself is running?

Reply:
I am not sure about this one. We need to weigh this against regular logging available for the containerd server process. Keeping it for now, till we have an answer or suggestion.